examples/first-frame-network/main.cpp

```diff
@@ -61,18 +61,43 @@ int main(int argc, char *argv[]) {
     }
 
     auto camera = cameras.front();
-    status = camera->initialize();
+
+    // user can pass any config.json stored anywhere in HW
+    status = camera->setControl("initialization_config",
+                                "config/config_walden_nxp.json");
     if (status != Status::OK) {
-        LOG(ERROR) << "Could not initialize camera!";
+        LOG(ERROR) << "Could not set the initialization config file!";
         return 0;
     }
 
+    status = camera->setControl("powerUp", "call");
+    if (status != Status::OK) {
+        LOG(ERROR) << "Could not PowerUp camera!";
+        return 0;
+    }
+
+    // optionally load configuration data from module memory
+    status = camera->setControl("loadModuleData", "call");
+    if (status != Status::OK) {
+        LOG(INFO) << "No CCB/CFG data found in camera module,";
+        LOG(INFO) << "Loading calibration(ccb) and configuration(cfg) data "
+                     "from JSON config file...";
+    }
+
     std::vector<std::string> frameTypes;
     camera->getAvailableFrameTypes(frameTypes);
     if (frameTypes.empty()) {
         std::cout << "no frame type available!";
         return 0;
     }
+
+    //set depthCompute to on or off
+    status = camera->setControl("enableDepthCompute", "on");
```

Comment:
enableDepthCompute is set to 'on' by default. Why was needed to be added here?

Reply:
removed